ariadne/cardano/test/backend/Test/Spec/AcidState.hs

```diff
@@ -0,0 +1,80 @@
+module Test.Spec.AcidState (spec) where
+
+import System.IO.Temp
+
+import Data.Acid (closeAcidState, openLocalStateFrom, query)
+import Data.Acid.Local (getState)
+
+import qualified Data.ByteString.Lazy as BL
+import Data.Text.Buildable (build)
+import Data.Text.Lazy.Builder (toLazyText)
+
+import Test.Hspec (Spec, describe)
+import Test.Hspec.QuickCheck (prop)
+import Test.QuickCheck (arbitrary, withMaxSuccess)
+import Test.QuickCheck.Monadic (monadicIO, pick, run)
+
+import Test.Spec.CreateWallet (NewWallet, applyNewWallet, genNewWalletRq)
+import Test.Spec.Fixture (genSpendingPasswords, withLayerLocalStorage)
+
+import Ariadne.Wallet.Cardano.Kernel.DB.AcidState (DB(..), Snapshot(..), defDB)
+import Ariadne.Wallet.Cardano.Kernel.DB.HdWallet (HdWallets(..))
+import Ariadne.Wallet.Cardano.Kernel.DB.Util.IxSet (IxSet, size)
+import Ariadne.Wallet.Cardano.Kernel.Internal (PassiveWallet(..), wallets)
+import Ariadne.Wallet.Cardano.WalletLayer (PassiveWalletLayer, pwlCreateWallet)
+
+spec :: Spec
+spec = describe "Checking AcidState related functions behavior" $ do
+  prop "Check empty DB state" $ withMaxSuccess 1 $

```

Comment:
```suggestion
  prop "can open empty database" $ withMaxSuccess 1 $
```

Reply:
My options are consistent with other descriptions of the tests.

Reply:
Ok, right. Thanks.